frontend/src/metabase/query_builder/components/filters/FilterPopover/FilterPopover.unit.spec.tsx

```diff
@@ -154,4 +129,46 @@ describe("FilterPopover", () => {
       });
     });
   });
+  describe("filter rendering", () => {
+    describe("no-value filters: %s filter", () => {
```

Comment:
`%s` is supposed to be in the `it.each` name. You can see that the test names are not using any of the specified options below e.g. `is-null` or `not-null`
![image](https://github.com/metabase/metabase/assets/1937582/7ecf1d0d-7a57-418e-8dd2-cfbe320ad4c1)

Reply:
Ah I pushed some changes but not others. Just need to push